src/services/challenge-issue/index.ts

```diff
@@ -168,6 +168,15 @@ export default class ChallengeIssueService {
       status: Status.Failed,
     };
 
+    // Check if is a pull request
+    if (pickUpQuery.issue?.isPullRequest) {
+      return {
```

Comment:
I think we should just ignore this command and no reply.

Reply:
Should we log for it?

Reply:
Yes, you can add some logs here.